Sources/Components/ARCTitleImageCard.swift

```diff
@@ -16,6 +16,13 @@ public struct ARCTitleImageCard<Trailing: View>: View {
 
     @ViewBuilder public var trailing: () -> Trailing
 
+    @Environment(\.horizontalSizeClass) var horizontalSizeClass
```

Comment:
Not sure if you saw my message [here](https://3sidedcube.slack.com/archives/C0276LH6CEN/p1687777447561689), but I don't think the designer's are quite using `horizontalSizeClass` as it was designed to be used.

So I think whenever a view should adapt for landscape we merely add
```swift
@Environment(\.verticalSizeClass) var verticalSizeClass
```

then for the conditional logic we have
```swift
verticalSizeClass == .regaular ? <#portrait#> : <#landscape#>
```

You can keep the `isPortrait` computed property if you really want but since it's just one small thing it might be over-kill